index.js

```diff
@@ -1,3 +1,6 @@
+require('@iopipe/trace');
+require('@iopipe/event-info');
```

Comment:
Why these requires?

Reply:
This repo counts on those deps being present for functionality to work correctly. If we import them here, we can make sure that users using webpack will have a bundle that will include those deps. We are only using `require` on `@iopipe/config` in `core`, so it makes sense that we should do the require of these deps here.